cabal.project.ci.windows

```diff
@@ -62,4 +68,16 @@ source-repository-package
   location: https://github.com/fossas/tar
   tag: 6bd13acc541ae60b5c831823243040e18bd1d957
 
+-- TODO: Add commentary after making a PR against upstream
+source-repository-package
+  type: git
+  location: https://github.com/fossas/codec-rpm
+  tag: 0f7431423d47fdf36945e4ff31fbee76005b7e68
+
+-- This can be removed once https://github.com/weldr/codec-rpm/pull/40 merges and is available on hackage.
+source-repository-package
+  type: git
+  location: https://github.com/fossas/codec-rpm.git
+  tag: 0f7431423d47fdf36945e4ff31fbee76005b7e68
+
```

Comment:
```suggestion
-- This can be removed once https://github.com/weldr/codec-rpm/pull/40 merges and is available on hackage.
source-repository-package
  type: git
  location: https://github.com/fossas/codec-rpm.git
  tag: 0f7431423d47fdf36945e4ff31fbee76005b7e68

```

Reply:
Thanks!